backend/app/views.py

```diff
@@ -136,7 +144,9 @@ def get_document(request, doc_id):
     """
     API Endpoint to get a document based on the ID
     """
-    doc_obj = Document.objects.get(id=doc_id)
+    queryset = Document.objects.all()
+    doc_obj = get_object_or_404(queryset, pk=doc_id)
```

Comment:
Per the [docs](https://docs.djangoproject.com/en/3.2/topics/http/shortcuts/#get-object-or-404), `get_object_or_404` can accept a Modal, Manager, or QuerySet. Not knowing the implementation under the hood, I'd be concerned about passing a QuerySet of `Document.objects.all()` and think we could simplify this by just passing the Model.